frontend/src/metabase/home/components/EmbedHomepage/InteractiveTabContent.tsx

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line no-restricted-imports
+import { List } from "@mantine/core";
+import { Link } from "react-router";
+import { t, jt } from "ttag";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Anchor, Button, Icon, Text } from "metabase/ui";
+
+import type { EmbedHomepageViewProps } from "./EmbedHomepageView";
+
+export const InteractiveTabContent = (props: EmbedHomepageViewProps) => {
+  const {
+    embeddingAutoEnabled,
+    interactiveEmbeddingQuickstartUrl,
+    exampleDashboardId,
+    licenseActiveAtSetup,
+    learnMoreInteractiveEmbedUrl,
+  } = props;
+  return (
+    <>
+      <Text
+        mb="md"
+        lh="1.5"
+      >{t`Use interactive embedding to offer multi-tenant, self-service analytics and dashboard creation in their own data sandbox. Pro and Enterprise plans only. `}</Text>
+
+      <Text fw="bold" mb="md">{t`The TL;DR:`}</Text>
+
+      <List
+        type="ordered"
+        mb="lg"
+        size="sm"
+        // TODO: fix this in a better way
+        style={{ listStyleType: "decimal" }}
+      >
+        {licenseActiveAtSetup === false && (
```

Comment:
I feel like booleans that could be undefined would be a bit awkward when the default value should be true. I'm not sure if that's the case, but the structure suggests so.

The reason sometimes we have `someBoolean === false` is to ensure `undefined` doesn't satisfy this clause. I feel like making the boolean that would have the default value as false would feel more natural. Since you can just do

```suggestion
        {!licenseActiveAtSetup && (
```

That being said, I guess from the name, this value should be `false` unless users activate their license, so I don't see the reason why we need to use `xxx === false`

Reply:
to elaborate on what I meant by the default value as false. That covers the case for `null` and `undefined` too. I should have said falsy value by default.

It would feel more natural if you pass `undefined`, `null`, or `false` and gets the same result. Because It gets awkward sometimes if you pass `undefined` and `false` and get different results.

Reply:
Ah I see, yeah that makes sense, I'll make sure not to render this when the any of the flags are undefined/null as it means they did the setup before this was introduced